app/helpers/application_helper.rb

```diff
@@ -31,4 +33,11 @@ def search_form_action_url(options = {})
 
     search_action_url(options)
   end
+
+  def render_thumbnail(document)
+    response = HTTParty.get(document.manifests.first)
+    return '' if response['thumbnail'].blank?
+
+    content_tag(:div, link_to(image_tag(response['thumbnail']['@id']), "/catalog/#{document.id}"), 'class': 'thumbnail', 'alt': "Thumbnail image for #{document._source['title_display']}")
+  end
```

Comment:
I am glad you are checking the response for errors, could we also add a `rescue` block to this method? It makes me uneasy than an HTTP exception fetching the thumbnail could prevent the page from rendering altogether.

Something like this:

```
def render_thumbnail(document)
  response = HTTParty.get(document.manifests.first)
  return '' if response['thumbnail'].blank?

  content_tag(:div, link_to(image_tag(response['thumbnail']['@id']), "/catalog/#{document.id}"), 'class': 'thumbnail', 'alt': "Thumbnail image for #{document._source['title_display']}")
rescue StandardError
  ''
end
```

Reply:
That's a really good point; I've adjusted the method, just waiting for tests to pass now.